app/Listeners/QueueMessage.php

```diff
@@ -33,20 +33,18 @@ public function handle(QueueMessageRequest $event)
         $resources = $event->resources;
 
         // Build the email.
-        $email = new Email($resources['message'], $resources['contest'], $resources['competition'], $resources['users']);
-
-        foreach ($email->allMessages as $message) {
-            $content = $message['message'];
+        $email = new Email($resources);
```

Comment:
@weerd @DFurnes yea but then how do i instantiate the `Manger` class before I pass it to the `Email` class? Can I do magical injection in the Listener construct?